kolibri/plugins/device/assets/src/views/ManageContentPage/SelectNetworkAddressModal/SelectAddressForm.vue

```diff
@@ -218,6 +218,8 @@
             this.savedAddresses = addresses;
             this.resetSelectedAddress();
             this.stage = this.Stages.FETCHING_SUCCESSFUL;
+            this.savedAddressesInitiallyFetched = true;
+            this.saved = true;
```

Comment:
What's `this.saved` for? I don't see it in the `data` object

Reply:
ah, I think that is a fluke mistake!

Reply:
I've deleted it